packages/clerk-js/src/ui/hooks/useInView.ts

```diff
@@ -0,0 +1,47 @@
+import React, { useCallback, useRef, useState } from 'react';
+
+import type { IntersectionOptions } from '../components/OrganizationSwitcher/OtherOrganizationActions';
```

Comment:
Can we move this type definition into this file to avoid importing from a specific component?

Reply:
I probably missed that, moving this line inside useInView